tests/FakeItEasy.Specs/CancellationSpecs.cs

```diff
@@ -149,7 +157,7 @@ public void AsyncNonCancelledCancellationToken(IFoo fake, CancellationToken canc
                 .x(() => cancellationToken = new CancellationToken(false));
 
             "When an async method is called with this cancellation token"
-                .x(() => task = fake.BarAsync(cancellationToken));
+                .x(() => { task = fake.BarAsync(cancellationToken); });

```

Comment:
Not a big deal, but what prompted this change (and its friends)?
It's a departure from many other of the one-line assignments in the file.